<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Firma Ekranı</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0451977 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-1</y>
      <width>401</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">QWidget#tab{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0451977 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));}</string>
     </property>
     <attribute name="title">
      <string>QR Oluştur</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>381</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Kaç TL lik QR Kod oluşturulacağını giriniz:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_tl">
         <property name="styleSheet">
          <string notr="true">
border:1px solid;
border-radius:5px;
background-color:white;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_qrcodeshow">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>165</y>
        <width>250</width>
        <height>250</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_qrolustur">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color:white;border:1px solid;</string>
      </property>
      <property name="text">
       <string>Oluştur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_qrbilgi">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>410</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cikis">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>5</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;border-color: black;border-width: 2px; border-style: outset;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true">QWidget#tab_2{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0451977 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));}</string>
     </property>
     <attribute name="title">
      <string>Satışlar</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>381</width>
        <height>371</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>3</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <item row="0" column="0">
       <property name="text">
        <string>AD</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>SOYAD</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>ADRES</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>MÜŞTERİ E POSTA</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>SATIŞ MİKTARI</string>
       </property>
      </item>
      <item row="0" column="5">
       <property name="text">
        <string>FİRMA E POSTA</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_firmabakiye">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>9</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cikis_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>5</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;border-color: black;border-width: 2px; border-style: outset;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_tl</tabstop>
  <tabstop>pushButton_qrolustur</tabstop>
  <tabstop>pushButton_cikis</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_cikis_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
